accounting/bank/reconciliation/reconciliation_models.rst

```diff
@@ -0,0 +1,122 @@
+---------------------------------------------
+
+When you are reconciling an entry with an *Open Balance*, you can use the buttons available under
+the *Manual Operations* tab to pre-fill all the values automatically, before validating the
+reconciliation. Each button is a different Reconciliation Model.
+
+.. image:: media/reconciliation_models_button.png
+   :align: center
+   :alt: Example of a Reconciliation Model with a write-off button in Odoo Accounting
+
+.. _reconciliation_models_suggestion:
+
+Suggest counterpart values
+--------------------------
+
+This type of Reconciliation Model suggests immediately counterpart values that only need to be
+validated. This automation is based on a set of rules defined in the reconciliation model.
+
+.. image:: media/reconciliation_models_suggestion.png
+   :align: center
+   :alt: Example of a Reconciliation Model that suggests counterpart values in Odoo Accounting
+
+.. _reconciliation_models_match:
+
+Match existing invoices/bills
+-----------------------------
+
+This type of Reconciliation Model selects automatically the right Customer Invoice or Vendor Bill
+that matches the payment. All that is left to do is to validate the entry. This automation is based
+on a set of rules defined in the reconciliation model.
+
+.. image:: media/reconciliation_models_match.png
+   :align: center
+   :alt: Example of a Reconciliation Model that matches existing invoices and bills automatically
+         in Odoo Accounting
+
+Configuration
+=============
+
+To manage or create new **Reconciliation Models**, go to :menuselection:`Accounting -->
+Reconciliation --> Reconciliation Models`. Alternatively, you can also open this menu from the
+Accounting Overview, by going to your Bank Journal car, clicking on the three little dots, and then
+on *Reconciliation Models*.
+
+.. image:: media/reconciliation_models_overview.png
+   :align: center
+   :alt: Open the Reconciliation Model menu from the overview dashboard in Odoo Accounting
+
+.. important::
+   The first entry, named *Invoices Matching Rule*, is the one responsible for the current matching
+   of invoices and bills. Therefore, it is advised to leave it at the top of the list and not to
+   delete it.
+
+Open the model you want to modify, or click on *Create* to create a new model, then fill out the
+form.
+
+Type
+----
+
+See :ref:`above <reconciliation_models_types>` for an explanation about the different types of
+Reconciliation Models.
+
+.. note::
+   If the *Documents* application is installed on your database, an additional **Activity type**
+   field appears when *To check* is ticked. Selecting the value *Reconciliation request* implies
+   that, whenever you use this model, a *Request Document* window pops up to request a document from
+   a user.
+
+Conditions on Bank Statement Line
+---------------------------------
+
+Define here all the conditions that are required for a Reconciliation Model to be applied.
+
```

Comment:
maybe a screenshot here?
I started looking at the screenshots above to see if I would find one showing the option, and I didn't find it.
But, I am illiterate on the subject (most readers are probably) so it might not be a valid suggestion.

Reply:
@kbr-odoo  I added screenshots for conditions and counterparts. Cheers :)